README.md

```diff
@@ -1,26 +1,40 @@
 # gistReferenceData
-Reference data curated by Semantic Arts, including units of measure, aspects, etc.
+The gist reference data includes units of measure and measurable characteristics, similar to [QUDT](https://qudt.org) but extended to include financial and process metrics and also modified to conform to a simpler ontology.
+
+## Basic terminology
+
+- A unit of measure is standard amount used to measure or specify things, like foot, yard, hour, watt, US dollar, etc.
+- An aspect is a measurable characteristic like cost, area, mass, etc. (called Quantity Kind in QUDT).
+- A unit group is a collection of units that can all be used measure the same aspect, e.g. the collection of units that measure mass.
 
 ## Data files
 
-- uomReferenceData.ttl  Complete set of unit of measure reference data.
-- uomReferenceData-starter-set.ttl A subset of the reference data that includes a few entries for each discipline.
-- currency.ttl Complete set of currencies extracted from QUDT.
-- currency-starter-set.ttl A small subset of currencies.
+- uomReferenceData.ttl is the complete set of unit of measure reference data.
+- currency.ttl is a set of currencies extracted from QUDT.
 
-- prefixes.ttl
-- templateForX.ttl where X is Aspect, UnitGroup, or UnitOfMeasure.
+- uomReferenceData-starter-set.ttl is a subset of the reference data that includes a few entries for each discipline.
+- currency-starter-set.ttl is a small subset of currencies.
+
+- prefixes.ttl includes all the prefixes used in the reference data.
 
 ## Setting up your own repository (starter set)
 
 - Make sure all the namespaces in the prefix.ttl file are defined in your repository.
-- Load the latest version of gist into its own named graph.
+- Load the latest version of [gist](https://www.semanticarts.com/gist) into its own named graph.
```

Comment:
It may be helpful to mention the latest version number here, and update it with subsequent gist releases, to remind us to verify that his repo is still compatible with the latest gist version, and to indicate that this repo is being actively maintained.